src/main/java/net/dv8tion/jda/api/events/automod/update/AutoModerationRuleActionsUpdateEvent.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.api.events.automod.update;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.AutoModerationRule;
+import net.dv8tion.jda.api.entities.automod.AutoModerationAction;
+import net.dv8tion.jda.api.entities.automod.AutoModerationField;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+/**
+ * Indicates that a {@link net.dv8tion.jda.api.entities.AutoModerationRule rule} updated its actions.
```

Comment:
Update events always need the cache to be enabled, otherwise JDA doesn't know what field has changed.
Would add that as a requirement to the documentation